README.md

```diff
@@ -1,2 +1,75 @@
-# todolist2
-Repo for To Do List activity week 2 Module 2
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: To Do List
+
+> Description the project:
+
+It's a simple HTML page list of To Do tasks. The list will be styled according to the specifications listed later in this README file. This simple web page will be built using webpack and served by a webpack dev server.
+
+## Requirements:
+
+- Set up a new project with webpack that is based on the webpack exercise you have already completed.
+- Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). The index.html file must be set as a template using the HTML Webpack Plugin.
+- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
+description [string].
+completed [bool].
+index: [number].
+- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
+On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
+- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.
+- All your source files (index.html, index.js and style.css) must be located in /src directory and your distribution files will be generated by webpack and served by webpack dev server from /dist folder.
+
+## Built With:
+
+- Major languages: Javascript, HTML, CSS
+- Frameworks: webpack
+- Technologies used: VSC, Github
+
+## Live Demo (if available)
+
+
+## Getting Started:
+
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```

Comment:
- [ ] Kindly delete unnecessary headings and add `Getting Started` instructions, because some people may not know how to get started with webpack application